copy_this/modules/fc/fcoxid2afterbuy/application/controllers/admin/fcafterbuy_actions.php

```diff
@@ -26,5 +25,58 @@ public function fcResetTransactionData() {
         $sMessage = $oLang->translateString('SHOP_MODULE_AFTERBUY_TRANSACTIONDATA_RESET');
         $oUtilsView->addErrorToDisplay(oxNew('oxException', $sMessage));
     }
-    
+
+    /**
+     * Download the OXID log
+     *
+     * @param void
+     * @return void
+     */
+    public function fcDownloadOxidLog() {
+        $sLogFile = getShopBasePath().'/log/oxideshop.log';
```

Comment:
This can be different in oxid5. In oxid5-shops there will be a seperate logfile for all afterbuy related stuff. I would suggest we need to unify this also for OXID6-Shops. Background: in oxid6 logger is used. Due to we have to be compatible to both shop versions, I would say we refactor all fcWriteToLog-Calls to place and fill logfiles without the usage of shop methods